qcodes/tests/drivers/test_ami430.py

```diff
@@ -424,3 +435,19 @@ def test_blocking_ramp_parameter(current_driver, caplog):
 
         assert len([mssg for mssg in messages if 'blocking' in mssg]) == 0
 
+
+@pytest.mark.parametrize('field_limit', [2, 2.2], ids=['integer', 'float'])
```

Comment:
Should we have tests with numpy integer and float data types just in case

Reply:
sure! shall i parametrize over all concrete and non-concrete types from qcodes/utils/types.py?

Reply:
Perhaps that is overkill but sounds good

Reply:
yes, it is an overkill... but i still did it, so let me push it as well...